packages/astro/src/vite-plugin-dev-toolbar/vite-plugin-dev-toolbar.ts

```diff
@@ -34,6 +38,18 @@ export default function astroDevToolbar({ settings, logger }: AstroPluginOptions
 					`Failed to initialize dev toolbar app ${args.app.name} (${args.app.id}):\n${args.error}`
 				);
 			});
+
+			server.ws.on('astro:devtoolbar:app:toggled', (args) => {
+				// Debounce telemetry to avoid recording events when the user is rapidly toggling apps for debugging
+				clearTimeout(telemetryTimeout);
+				telemetryTimeout = setTimeout(() => {
+					telemetry.record(
+						eventCliSession('dev', settings.config, {
+							app: args?.app?.id ?? 'unknown',
```

Comment:
Should we anonymize this? Just make it so we know when people are using Astro apps and otherwise send "unknown"?